replication/time_test.go

```diff
@@ -49,3 +49,22 @@ func (s *testSyncerSuite) TestTime(c *C) {
 		c.Assert(formatZeroTime(t.frac, t.dec), Equals, t.expected)
 	}
 }
+
+func (s *testTimeSuite) TestTimeStringLocation(c *C) {
+	t := fracTime{
+		time.Date(2018, time.Month(7), 30, 10, 0, 0, 0, time.FixedZone("EST", -5*3600)),
```

Comment:
do you need to use field name here?

Reply:
Don't need to if you specified everything.